### DISCUSSION TITLE
Move freeverb@ and gverb@ to existing gen~ implementations in Max documentation

### DISCUSSION THREAD

**Discussion by tobinja1** [ppooll-dev/ppooll#144] [Ideas]
Just an idea to cut down on external dependencies and do a bit of future-proofing. Max comes with both freeverb and gigaverb implementations in its existing documentation, I could personally handle the configuration to get the acts functioning identically!

**Comment by chausch** [comment]
> Just an idea to cut down on external dependencies and do a bit of future-proofing. Max comes with both freeverb and gigaverb implementations in its existing documentation, I could personally handle the configuration to get the acts functioning identically!

sounds great! however, the freeverb gen~ implementation does not have the freeze function that the external features. would be great if the dsp code for that can be tracked down somewehere and added. do you think you could find that somewhere?

**Reply by tobinja1** [reply]
Actually, I wrote an audio stutterer in gen~ that does more-or-less the same thing a few years back, I'll go track it down and see if I can get it to play nice with freeverb's existing gen~ code!
